client/src/utils/urlify.test.js

```diff
@@ -9,6 +9,36 @@ describe('urlify', () => {
         'lisboa-e-otima-a-beira-mar',
       );
     });
+
+    it('should keep leading spaces & convert to hyphens if supplied', () => {
+      expect(urlify('  I like _ßpaces')).toBe('-i-like-_sspaces');
+    });
+
+    it('should allow a supplied locale to re-order downcode mapping', () => {
+      // mix of characters that are transliterated differently across Russian/Ukrainian
+      const chars = 'Ц-йи-Ґ-Х-щ';
+      const ruTransliteration = 'c-ji-g-x-shh';
+      const ukTransliteration = 'ts-iy-g-kh-shch';
+      expect(urlify(chars)).toEqual(ruTransliteration); // by default the Russian values come through with priority
+      expect(urlify('Ц-йи-Ґ-Х-щ', { locale: 'ru' })).toEqual(ruTransliteration);
+      expect(urlify('Ц-йи-Ґ-Х-щ', { locale: 'uk' })).toEqual(ukTransliteration);
+
+      // another example
+      expect(urlify('Георгій', { locale: 'uk-UK' })).toEqual('heorhii');
```

Comment:
In my opinion it is not quite right to take a word in one language and see how they differ in transliteration with different localization. In this case I would use another name (not Георгий(ru)/Георгій(uk) but Богдан(ru/uk)), because it is spelled the same in both languages and is not misleading.

Reply:
Great. I'll update this unit test. Thank you.

Reply:
@ACK1D any chance you have an example I can use that's 'real world ish' for the Russian/ Belarusian difference. I'd like to add a unit test for that also.

Reply:
> @ACK1D any chance you have an example I can use that's 'real world ish' for the Russian/ Belarusian difference. I'd like to add a unit test for that also.

As an option to check the Russian sentence for transliteration in the Belarusian localization (in theory it should break because of the absence of the letters “И” and “Щ” in the language, I think for these cases it is necessary to take the Russian localization. I mean in priority the Belarusian localization, but if some letters are missing, then still look at the Russian version, because in Belarus there are two official languages Russian and Belarusian).
Sentence: "Бен делает отличные вещи для сообщества трясогузки!"

Reply:
ohh very kind, I have refined it a bit but will use this. Thank you.